drivers/led/Kconfig.ht16k33

```diff
@@ -0,0 +1,15 @@
+#
+# Copyright (c) 2019 Henrik Brix Andersen <henrik@brixandersen.dk>
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+menuconfig HT16K33
+	bool "HT16K33 LED driver with keyscan"
```

Comment:
This driver is not handling keyscan functionality, so no need to mention here and in driver.

Reply:
@Mani-Sadhasivam That's actually the name of the chip, but I can see how that can be confusing. I have removed the mention of the keyscan from here.